packages/styles/exports.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
We should look into if these files can be automatically generated - it might be easy to miss an update here if files are changed upstream.